details.php

```diff
@@ -0,0 +1,53 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+
+    <title>ARTIST WEBSITE</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ARTIST WEBSITE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lobster&family=Oswald:wght@200..700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="stylesheet.css">
+    <script src="https://kit.fontawesome.com/a767a9281d.js" crossorigin="anonymous"></script>
+</head>
```

Comment:
Because this is the same on every page, you can actually create a `header.php` file, then do `require_once 'header.php'` to use it wherever it's needed.